src/components/pages/create-account/RecaptchaNotice.jsx

```diff
@@ -12,6 +13,11 @@ export const RecaptchaNotice = () => (
         rel="noopener noreferrer"
         target="_blank"
       >
+        <Icon
+          alt="Icône lien vers site externe"
```

Comment:
```suggestion
          alt=""
```
Sinon le lecteur d'écran va lire "Icône lien vers site externe Charte des Données Personnelles"

Reply:
Par contre ça vaut le coup de préciser dans le title que c'est un lien externe, parce que le "Charte machin bidule" ne le précise pas

Reply:
Exemple: https://www.service-public.fr/ bas de page (twitter / facebook)

Reply:
Ok, c'était tout à fait voulu de mettre un alt sur l'icône pour préciser que le lien à venir menait vers un site externe, mais il est peut-être plus judicieux de le préciser dans un title comme le dit Flora

Merci